### ISSUE TITLE
ansible-vault 2.0.0.2 fails to decode secure file with accented characters

### ISSUE THREAD

**Issue by kblcuk** [ansible/ansible#14258]
##### Issue Type:
- Bug Report

May be related to [some of these](https://github.com/ansible/ansible/issues?utf8=✓&q=is%3Aissue+is%3Aopen+ordinal+is)? 
##### Ansible Version:

```
ansible --version
ansible 2.0.0.2
  config file = /Users/kblcuk/manyn/ansible.cfg
  configured module search path = Default w/o overrides
```
##### Ansible Configuration:

```
$ cat ~/manyn/ansible.cfg
# config file for Ansible -- http://ansible.com/
# ==============================================

[defaults]
# Add our community_roles directory
roles_path = ansible/community_roles

vault_password_file=~/.vault-pass

callback_plugins = ansible/plugins/callbacks/
```
##### Environment:

Latest OSX, but not sure if this is OS-related
##### Summary:

If encoded file contains any non-ascii (?) characters, viewing that file fails.
Strangely, you can still edit it.
##### Steps To Reproduce:

```
ansible-vault create test_secure.yml
```

Add something like

```
foo: bår
```

Try

```
ansible-vault view test_secure.yml

##### Expected Results:
Shows the file

##### Actual Results:
```

$ ansible-vault view test_secure.yml
Unexpected Exception: 'ascii' codec can't decode byte 0xc3 in position 6: ordinal not in range(128)
to see the full traceback, use -vvv

```
$ ansible-vault view test_secure.yml -vvv
Using /Users/alex/Code/operations/ansible.cfg as config file
Unexpected Exception: 'ascii' codec can't decode byte 0xc3 in position 6: ordinal not in range(128)
the full traceback was:

Traceback (most recent call last):
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/bin/ansible-vault", line 85, in <module>
    sys.exit(cli.run())
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/lib/python2.7/site-packages/ansible/cli/vault.py", line 119, in run
    self.execute()
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/lib/python2.7/site-packages/ansible/cli/__init__.py", line 99, in execute
    fn()
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/lib/python2.7/site-packages/ansible/cli/vault.py", line 160, in execute_view
    self.pager(self.editor.plaintext(f))
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/lib/python2.7/site-packages/ansible/cli/__init__.py", line 449, in pager
    self.pager_pipe(text, os.environ['PAGER'])
  File "/usr/local/Cellar/ansible/2.0.0.2/libexec/lib/python2.7/site-packages/ansible/cli/__init__.py", line 462, in pager_pipe
    cmd.communicate(input=text.encode(sys.stdout.encoding))
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 6: ordinal not in range(128)
```

**Comment by abadger** [comment]
Just tested your issue and this should be fixed in the upcoming 2.0.1 release.  Here's the commit to the stable-2.0 branch where it was fixed: 9ddb4a985c86684100bef85df25088957283e963

**Comment by kblcuk** [comment]
Awesome! Thanks.
